### PR TITLE
Add withdrawn to student intervention and outreach models

### PR DESCRIPTION
**Pull request by dting**

### CODE REVIEW

File: client/app/main/records/upload/upload.controller.js [hunk 1]
```diff
@@ -103,7 +104,8 @@ function UploadCtrl($scope, PDF, AbsenceRecord, Auth, School, toastr) {
           absences: trigger.absences,
           student: entry.student,
           school: school._id,
-          schoolYear: schoolYear
+          schoolYear: schoolYear,
+          withdrawn: !!(prevEntry.student || {}).withdrawn
         };
```
Comment (pdotsani) [inline client/app/main/records/upload/upload.controller.js]: Wouldn't this be the same as declaring `(prevEntry.student || {}).withdrawn`?

**Comment by pdotsani** [comment]
lgtm.

File: client/app/main/records/upload/upload.controller.js (hunk 1 above)
Comment (dting) [inline client/app/main/records/upload/upload.controller.js]: I'm casting it to a boolean. It is the same as doing `Bool((prevEntry.student || {}).withdrawn)`.

Comment (pdotsani) [inline client/app/main/records/upload/upload.controller.js]: Gotcha. Thanks for the clarification.

**Pull request merged**